packages/db/src/migrations/20240614185515_populate_member_id_in_admins_table.ts

```diff
@@ -0,0 +1,18 @@
+import { type Kysely } from 'kysely';
+
+export async function up(db: Kysely<any>) {
+  // Populate member_id column in admins table
+  await db
+    .updateTable('admins')
+    .set({
+      member_id: db
+        .selectFrom('students')
+        .select(['id'])
+        .where('students.email', '=', 'admins.email'),
+    })
+    .execute();
+}
```

Comment:
We do want to try to have just one migration per PR, so it's fine to combine this with the `up` function in the other migration (and similarly with the `down` function)